test/auth_cluster/test_auth_v2_migration.py

```diff
@@ -184,6 +184,7 @@ async def test_auth_v2_migration(request, manager: ManagerClient):
 
 @pytest.mark.asyncio
 async def test_auth_v2_during_recovery(manager: ManagerClient):
+    # FIXME: move this test to the new Recovery procedure or remove it if unneeded.
```

Comment:
Gleb made a good point: instead of new/old, name it gossip-topology and raft-topology, or something like that